bean/internal/usecase/passwordless/passwordless.go

```diff
@@ -92,10 +83,27 @@ func (u *UseCase) findOrCreateUser(email string) (*model.User, error) {
 		return user, nil
 	}
 
-	user, err = u.users.Create(email)
+	user, err = u.Users.Create(email)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create user: %w", err)
 	}
 
 	return user, nil
 }
+
+func buildEmailBody(tokenID, password string) string {
+	return fmt.Sprintf(
+		("Hello." +
+			"\r\n\r\n" +
+			"Use this link to login to Bean:" +
+			"\r\n" +
+			"https://localhost:8080/login?i=%s&p=%s" +
+			"\r\n\r\n" +
+			"This link will expire in 10 minutes." +
+			"\r\n" +
+			"If you did not request this, don't worry." +
+			"\r\n\r\n" +
+			"Cheers."),
+		tokenID, password,
+	)
+}
```

Comment:
This should be moved to a template

Also we should be passing the base URL and route

It's acceptable for now though